portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -58,6 +62,8 @@ function onLoad() {
   ];
 
   loadMap();
+  insertSearch();
+  authenticationStatus();
```

Comment:
You could probably move these to the bottom or just the search as its unrelated to the Map.